.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -28,63 +24,59 @@ body:
     id: actual
     attributes:
       label: Actual outcome
-      description: Please provide the output produced by the above code, which may be a screenshot, console output, etc. 
-      placeholder: if applicable, paste the console output and images here
+      description: The output produced by the above code; e.g. a screenshot, console output, etc.
+      placeholder: Paste the console output and images here
     validations:
       required: true
   - type: textarea
     id: expected
     attributes:
       label: Expected outcome
-      description: Please provide a description (or visual example) of the expected outcome from the code snippet. If this used to work in an earlier version of Matplotlib, please note the version it used to work on
+      description: Description (or visual example) of the expected outcome from the code snippet.
     validations:
       required: true
+  - type: textarea
+    id: details
+    attributes:
+      label: Detailed description
+      description: |
+        Context, boundary conditions, etc. Has this worked in earlier versions?
+        If possible: Root cause, suggested fix.
   - type: input
     id: operating-system
     attributes:
       label: Operating system
```

Comment:
was gonna open a PR for this, but since you're tweaking - we should be consistent about capitalization in one direction or the other (all caps, sentence case)